src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -17,19 +17,25 @@ import store from '../../data/store';
 import TranscriptLanguage from './TranscriptLanguage';
 
 import { DATE_FORMAT, IN_REVIEW_STATUS, PUBLISHED } from '../../data/constants';
-import { endDateHelp, startDateHelp } from '../../helpText';
-
+import { endDateHelp, startDateHelp, pacingHelp } from '../../helpText';
 
 const formatCourseRunTitle = (courseRun) => {
   if (courseRun) {
     const labelItems = [];
+    const publishDate = [];
     if (courseRun.start) {
       labelItems.push(moment.utc(courseRun.start).format('MMM Do YYYY'));
     }
     if (courseRun.pacing_type) {
       labelItems.push(courseRun.pacing_type.split('_').map(pacingType =>
         pacingType.charAt(0).toUpperCase() + pacingType.slice(1)).join(' '));
     }
+    if (courseRun.go_live_date) {
+      publishDate.push('is ');
+      publishDate.push(moment.utc(courseRun.go_live_date).format('MMM Do YYYY'));
```

Comment:
Might be best here to just have the fully interpolated string instead of constructing it with the array and `.join()`. This'll probably be hard to internationalize as is